go.mod

```diff
@@ -49,3 +49,13 @@ require (
 	gopkg.in/fsnotify.v1 v1.4.7
 	gopkg.in/yaml.v2 v2.2.2
 )
+
+replace (
```

Comment:
Do you have an actual explanation of why this is needed? Also, those versions are from where?

Also I  think Thanos don't depend on all of them (e.g `klog`?) (:

Reply:
This overrides the dependencies to use the exact commits of client-go instead of the sem ver that Go modules would pick, as those are broken.

We might not need klog, however it's a dependency of client-go so it's probably better to pin it in that case.

Reply:
ok, fair: https://github.com/thanos-io/thanos/blob/master/go.sum#L616

Can we move back to `proxy.golang.org` maybe as our Proxy? As agreed then?